assets/js/utils/rtcUtils.ts

```diff
@@ -17,7 +21,7 @@ export type MergedScreenRef = {
   refreshId: NodeJS.Timeout | undefined;
 };
 
-export const presenterStreams: { [key: string]: MediaStream } = {};
+export const presenterArea: { [key: string]: MediaStream } = {};
```

Comment:
Yeah, since this component contains preview, which isn't connected to RtcEngine, and no data is received from it I wouldn't say there are any *Streams* there.